tests/Jellyfin.MediaEncoding.Tests/Probing/ProbeResultNormalizerTests.cs

```diff
@@ -18,6 +18,19 @@ public class ProbeResultNormalizerTests
         private readonly JsonSerializerOptions _jsonOptions = JsonDefaults.Options;
         private readonly ProbeResultNormalizer _probeResultNormalizer = new ProbeResultNormalizer(new NullLogger<EncoderValidatorTests>(), null);
 
+        [Theory]
+        [InlineData("2997/125", 23.976f)]
+        [InlineData("1/50", 0.02f)]
+        [InlineData("25/1", 25f)]
+        [InlineData("120/1", 120f)]
+        [InlineData("1704753000/71073479", 23.98578237601117f)]
+        [InlineData("0/0", 0f)]
```

Comment:
I think 0 is the preferred output instead of `null`, but I'm not 100%

Reply:
I disagree. Division by 0 is undefined. I would expect it to return null.